examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
I am planning to move benchmarks outside the library, as a separate repo/module in some future, since I will be also providing benchmarks for UME::Vector. It would be neat to have them all in one place and sharing build infrastructure. But that is somthing that should be postponed.

I'll try to make the aggregate repo, so that the hierarchy gets clearer.